src/Development/IDE/Core/PositionMapping.hs

```diff
@@ -46,19 +46,23 @@ toCurrent (Range (Position startLine startColumn) (Position endLine endColumn))
     | line > endLine || line == endLine && column >= endColumn =
       -- Position is after the change so increase line and column number
       -- as necessary.
-      Just $ Position (line + lineDiff) newColumn
+      Just $ newLine `seq` newColumn `seq` Position newLine newColumn
```

Comment:
Do you want `$!` here to force the argument to `Just`? Or alternatively `seq` outside of the `Just` application.

I’m a bit sad `Position` isn’t strict in its field.

Reply:
I'll go with `$!`, thanks!